### PR TITLE
Add mentor notes for java/secret-handshake

### PR DESCRIPTION
**Pull request by jmrunkle**
This is just a start, and anyone with better ideas is welcome to amend it.

### CODE REVIEW

File: tracks/java/exercises/secret-handshake/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            }
+        }
+        return result;
+    }
+}
+```
+
+### Common suggestions
+
+- TODO: add some common suggestions here
```
Comment (SleeplessByte) [inline tracks/java/exercises/secret-handshake/mentoring.md]: Remove TODO

File: tracks/java/exercises/secret-handshake/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+### Talking points
```
Comment (SleeplessByte) [inline tracks/java/exercises/secret-handshake/mentoring.md]: Remove heading

File: tracks/java/exercises/secret-handshake/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+- TODO: additional suggestions
```
Comment (SleeplessByte) [inline tracks/java/exercises/secret-handshake/mentoring.md]: Remove TODO

**Comment by jmrunkle** [comment]
@SleeplessByte - is there something I have to do other than the patch? I do not know how to change assignees (or if I even can).

**Comment by SleeplessByte** [comment]
@jmrunkle Nope, now it's just waiting! Let's ping @exercism/java / @Smarticles101 directly; if it stays like this for another week we'll just merge it because having some notes is better than none. (cc @rpottsoh)

**Pull request merged**